example/ios/Podfile.lock

```diff
@@ -10,8 +10,8 @@ PODS:
     - React-Core (= 0.66.1)
     - React-jsi (= 0.66.1)
     - ReactCommon/turbomodule/core (= 0.66.1)
-  - Fidel (2.0.0-beta8)
-  - fidel-react-native (2.0.0-beta4):
+  - Fidel (2.0.0-beta9)
+  - fidel-react-native (2.0.0-beta6):
```

Comment:
Why this moves from `2.0.0-beta4` to `2.0.0-beta6` and not `2.0.0-beta5`?

Reply:
@davi-mezencio a fresh install will always use the latest NPM version, which is `2.0.0-beta6`. Since this is the scenario that triggered the error for our customer, there is no reason to not use the latest version.